src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -567,23 +545,23 @@ bool ColorControlServer::shouldExecuteIfOff(EndpointId endpoint, uint8_t optionM
     // ---------- The following order is important in decision making -------
     // -----------more readable ----------
     //
-    if (optionMask == 0xFF && optionOverride == 0xFF)
+    if (optionMask == static_cast<OptionsBitmap>(0xFF) && optionOverride == static_cast<OptionsBitmap>(0xFF))
```

Comment:
Followup (separate PR): we should properly express "not present" here (e.g. via Optionals).